setup.py

```diff
@@ -69,7 +69,9 @@
 install_requires = [
     'Flask>=0.11.1',
     'pyPEG2>=2.15.2',
-    'elasticsearch>=2.0.0,<3.0.0',  # FIXME unpin for Elastic 5.0
+    # NOTE: Overlays have to choose which elasticsearch version they want to
+    # use and pin both elasticsearch and elasticsearch-dsl libraries.
+    'elasticsearch>=2.0.0',
     'elasticsearch-dsl>=2.0.0',
```

Comment:
We had a similar issue with B2Share (#88). Pinning the version in the overlay solved the issue as it constrained to version 2.x. Later when B2Share will migrate to ES 5.x we can just pin it to the next version, as long as the invenio modules support this version of the libs and ES of course.
We don't force a specific version of the SQL databases, maybe I am missing something but I would try to have the same flexibility for Elasticsearch.